packages/@azure/arm-databox/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@azure/arm-databox",
+  "author": "Microsoft Corporation",
+  "description": "DataBoxManagementClient Library with typescript type definitions for node.js and browser.",
+  "version": "0.0.1",
```

Comment:
Where did this version number come from?

Reply:
Make sure that the swagger specification didn't have a version-number property in it, please.